pkg/functions/functions.jsonnet

```diff
@@ -3,7 +3,9 @@
 
   local defaultIgnores = [
     'status.conditions',
-    'spec.writeConnectionSecretToRef'
+    'spec.writeConnectionSecretToRef',
+    'spec.forProvider.tags',
+    'spec.forProvider.tagging'
```

Comment:
we need to add `spec.forProvider.tagSpecifications`

check:
https://doc.crds.dev/github.com/crossplane-contrib/provider-aws/ec2.aws.crossplane.io/TransitGateway/v1alpha1@v0.32.0#spec-forProvider-tagSpecifications